packages/squiggle-lang/src/fr/list.ts

```diff
@@ -293,7 +300,10 @@ export const library = [
     examples: [`List.sortBy([{a:3}, {a:1}], {|f| f.a})`],
     definitions: [
       makeDefinition(
-        [frArray(frGeneric("A")), frLambdaTyped([frGeneric("A")], frNumber)],
+        [
+          frArray(frGeneric("A")),
+          frNamed("fn", frLambdaTyped([frGeneric("A")], frNumber)),
```

Comment:
I named most of the functions "fn", to be a bit more explicit/redundant. Not totally necessary though.